docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -31,6 +31,9 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
 NOTE: If you just launch `mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create` the Maven plugin asks
 for user inputs. You can disable (and use default values) this interactive mode by passing `-B` to the Maven command.
 
+NOTE: It is recommended you match gradle version you use with the version which the Quarkus Gradle Plugin is built with:
```

Comment:
I'm not sure about that. It's recommended to use `./gradlew` so that you don't have to care about anything :).

Reply:
@gsmet that is assuming you use the project generator. In my use case I don't use that because I like using the gradle kotlin dsl which the generator doesn't support. Looking at the documentation I had no idea what gradle version to use.

Reply:
OK. I think we probably need a Gradle version attribute similar to what we have with the Maven version.

I'll try to prepare something along those lines and amend your proposal.

Thanks!